tests/CMakeLists.txt

```diff
@@ -91,18 +87,6 @@ if (NOT LIBIA2_AARCH64)
     add_subdirectory(two_shared_ranges)
     add_subdirectory(trusted_indirect)
 
-    if(IA2_LIBC_COMPARTMENT)
-        add_subdirectory(libc_default_compartment)
```

Comment:
Did you mean to drop these tests?

Reply:
yes they were added above, this was redundant

Reply:
nvm, I see now that they were previously duplicated